### PR TITLE
AP_Scripting: pause quicktune during pilot input

### PR DESCRIPTION
**Pull request by tridge**
thanks to Marco for the suggestion

### CODE REVIEW

**Comment by robustini** [comment]
Thanks @tridge, work superfine!
I would add a message in the MESSAGES tab indicating the pause and restart of the procedure when moving the sticks and when re-centering.
Also note that the throttle is not contemplated, if it is necessary to run such LUA in Q_Loiter I would put that as well (center stick), but then it would no longer work in other flight mode, so it better not be contemplated.

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 1]
```diff
@@ -259,6 +259,7 @@ singleton SRV_Channels method set_angle void SRV_Channel::Aux_servo_function_t'e
 singleton SRV_Channels method set_range void SRV_Channel::Aux_servo_function_t'enum SRV_Channel::k_none SRV_Channel::k_nr_aux_servo_functions-1 uint16_t 0 UINT16_MAX
 
 ap_object RC_Channel method norm_input float
+ap_object RC_Channel method norm_input_dz float
```
Comment (IamPete1) [inline libraries/AP_Scripting/generator/description/bindings.desc:262]: needs docs

Comment (tridge) [reply libraries/AP_Scripting/generator/description/bindings.desc:262]: fixed

**Comment by tridge** [comment]
> I would add a message in the MESSAGES tab indicating the pause and restart of the procedure when moving the sticks and when re-centering.

In normal autotune that is needed as it greatly impacts throttle, which typically isn't sprung, so people often get it a bit off. This change only involves roll, pitch, yaw, which typically have springs, so less likely to be something the pilot isn't aware of

**Pull request merged**